### PR TITLE
ricochet programming defined

### PR DESCRIPTION
**Pull request by pawelpluta**

### CODE REVIEW

File: methodologies/ricochet-programming/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## When to use it
+
+Unfortunately, ricochets itself and Ricochet Programming is not something, that you can intentionally use. 
+They just happen, however, you can take steps and help them to occur. As this technique is relatively cheap, you should 
+aim to take those steps and try to trigger ricochets.
+
+## How to introduce
+
+The best in this methodology is that you can introduce it by yourself, not even telling anyone from it. However, consider
```
Comment (bandrzejewski) [inline methodologies/ricochet-programming/README.md]: , not evet telling anyone **about** it.

File: methodologies/ricochet-programming/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+The subtle, but very powerful technique, allowing you to boost up the development process and save your time for another round of Tekken 3.
+Ricochets are specialized tools for resolving small tasks and even user stories at almost zero cost. The smaller tasks are,
+the bigger benefit can be achieved. Ricochets are not very accurate, therefore they cannot be always 
+used, but when there are multiple targets, the probability of hitting increases.
+
+Let's understand the process of triggering ricochets:
+![Ricochet Programming overview](images/1-ricochet-overview.png)
+Assuming, that you are refining work pretty well, and create tasks for each user story. According to the timeline, developer 
+started the work from `Task A`. Once finished, `Task C` was started, and then `Task B`. During work on each of the tasks, 
+small pieces of code were added, that developer is unaware they might reflect other parts of code. Those ricochets, when
```
Comment (bandrzejewski) [inline methodologies/ricochet-programming/README.md]: maybe instead of 'small pieces of code' - 'smart **portions** of code'?

Comment (pawelpluta) [reply methodologies/ricochet-programming/README.md]: Good clarification. Done

File: methodologies/ricochet-programming/README.md (hunk 1 above)
Comment (pawelpluta) [reply methodologies/ricochet-programming/README.md]: Missed that.

**Review by bslota — APPROVED** [review]

**Pull request merged**